coordinape/coordinape
# migrate createUser FE to hasura

Fixes #630 


Test plan:

1. navigate to circle admin page
2. create some users
3. they should show up in the user list with the correct settings
